include/odp/api/spec/ipsec.h

```diff
@@ -238,6 +238,10 @@ typedef struct odp_ipsec_capability_t {
 	 */
 	odp_support_t retain_header;
 
+	/** Implementation will automatically truncate TFC padding in received
+	 *  packets */
+	odp_bool_t tfc_padding_truncate;
```

Comment:
Is the tfc_padding_truncate capability even needed? It is an optimization for applications that would benefit from knowing in advance that the padding is always truncated, but is there a good use case for that?

Reply:
@JannePeltonen 
Basically it is there to describe hardware behaviour. NXP SEC truncates packets according to L4 length, our hardware does not.

Reply:
Yes, but is it needed? With this, an application could check if the capability was set instead of checking if packet length is bigger than the header indicates for deciding whether to trim padding. I just wonder if that is beneficial enough to justify yet another capability flag.

Reply:
This will allow fast-path forwarding for some applications. If this flag is set, an application can skip header checks/lookups and forward ready packet.

Reply:
I still vote for minimal TFC specific API changes at this point. This capa is not strictly needed. A portable application would need to implement the length checks anyway.

Reply:
Fine, I'll drop this and add a phrase that an implementation might have removed the padding.